src/coordination/coordinator_state.cpp

```diff
@@ -36,6 +36,7 @@ CoordinatorState::CoordinatorState(ReplicationInstanceInitConfig const &config)
       .port = static_cast<uint16_t>(config.management_port),
   };
   data_ = CoordinatorMainReplicaData{.coordinator_server_ = std::make_unique<CoordinatorServer>(mgmt_config)};
+  spdlog::trace("Created coordinator server.");
```

Comment:
Like it but expand so we can also find out on which coordinator server, bolt server etc. I would print whole config